plugin/config.json

```diff
@@ -48,12 +48,7 @@
 		"Devices": null
 
 	},
-	"Env": [ {
-      "name": "VDVS_LOG_LEVEL",
-      "description": "Log level - info, warn, error or debug",
-      "value": "info",
-      "Settable": [ "value"]
-	},
+	"Env": [
 	{"name": "VDVS_DISCOVER_VOLUMES",
```

Comment:
You took out wrong config option. Keep VDVS_LOG_LEVEL  and remove VDVS_DISCOVER_VOLUMES